docs/kb/troubleshoot-functions.md

```diff
@@ -5,178 +5,109 @@ title: VMware Event Broker Function Troubleshooting
+vmware-functions     sockeye-trigger-dispatcher-7f4dbd7f78-n589p       1/1     Running     0          14h
+vmware-system        cadvisor-sk4j9                                    1/1     Running     0          14h
+vmware-system        tinywww-dd88dc7db-dqnnc                           1/1     Running     0          14h
+vmware-system        veba-rabbit-server-0                              1/1     Running     0          14h
+vmware-system        veba-ui-54967b4bf4-lpjrn                          1/1     Running     0          14h
+vmware-system        vmware-event-router-vcenter-6b76959df5-6mrb4      1/1     Running     3          14h
+vmware-system        vmware-event-router-webhook-6b48cc5b8c-sjzx8      1/1     Running     0          14h
+```
+
+First, we want to see if the event router is capturing events and forwarding them on to a function.
 
 Use this command to follow the live Event Router log.
 
 ```bash
-kubectl logs -n vmware vmware-event-router-5dd9c8f858-9g44h --follow
+kubectl logs -n vmware-system vmware-event-router-vcenter-6b76959df5-6mrb4 -f
 ```
 
-For this sample troubleshooting, we have the sample hostmaintenance alarms function running. To see if the appliance is properly handling the event, we put a host into maintenance mode. 
+For this sample troubleshooting, we have the sample [PowerCLI Tagging function](https://github.com/vmware-samples/vcenter-event-broker-appliance/tree/master/examples/knative/powercli/kn-pcli-tag) running which will react to a VM powered on Event (`DrsVmPoweredOnEvent`). To see if the appliance is properly handling the event, we create a test VM and power it on.
 
-When we look at the log output, we see various entries regarding EnteredMaintenanceModeEvent, ending with the following:
+When we look at the log output, we see various entries regarding DrsVmPoweredOnEvent, ending with the following:
 
 ```
-[OpenFaaS] 2020/03/11 22:15:09 invoking function(s) on topic: EnteredMaintenanceModeEvent
-[OpenFaaS] 2020/03/11 22:15:09 successfully invoked function powercli-entermaint for topic EnteredMaintenanceModeEvent
+2021-09-24T13:57:05.015Z	INFO	[KNATIVE]	knative/knative.go:181	sending event	{"eventID": "c6d61d55-8100-459e-a1e7-7a936bac6e43", "subject": "DrsVmPoweredOnEvent"}
+2021-09-24T13:57:05.015Z	INFO	[KNATIVE]	knative/knative.go:193	successfully sent event	{"eventID": "c6d61d55-8100-459e-a1e7-7a936bac6e43"}
+2021-09-24T13:57:06.017Z	INFO	[VCENTER]	vcenter/vcenter.go:343	invoking processor	{"eventID": "87af3e86-6516-4377-9a9b-86c4c9b00b05"}
 ```
 
 This lets us know that the function was invoked. If we still don't see the expected result, we need to look at the function logs.
 
-Each OpenFaaS function will have its own pod running in the openfaas-fn namespace. We can examine the logs with the following command.
+Each Knative function will have its own pod running in the vmware-functions namespace. We can examine the logs with the following command.
```

Comment:
Can you please add a note that the pod name has to be replaced by the actual name in the cluster? One comment is enough, i.e. not need to repeat for all log operations.